### PR TITLE
[cli] Updated to the latest changes in Host Orchestrator web API.

### PR DESCRIPTION
**Pull request by ser-io**
- Host Orchestrator changed its response object for create cvd requests.

### CODE REVIEW

File: pkg/cli/cli.go [hunk 1]
```diff
@@ -569,18 +569,23 @@ func runCreateCVDCommand(c *cobra.Command, flags *CreateCVDFlags, opts *subComma
...
 	if err != nil {
 		return err
 	}
-	statePrinter.Print(connectCVDStateMsg)
-	cvd.ConnStatus, err = ConnectDevice(flags.CreateCVDOpts.Host, cvd.Name, &command{c, &flags.Verbose}, opts)
-	statePrinter.PrintDone(connectCVDStateMsg)
-	if err != nil {
-		err = fmt.Errorf("Failed to connect to device: %w", err)
+	if len(cvds) > 0 {
+		cvd := cvds[0]
```
Comment (jemoreira) [inline pkg/cli/cli.go:578]: why connect only the first device? why not all?

Comment (ser-io) [reply pkg/cli/cli.go]: connected to all, wanted your feedback here.

File: pkg/cli/cli.go [hunk 2]
```diff
@@ -569,18 +569,22 @@ func runCreateCVDCommand(c *cobra.Command, flags *CreateCVDFlags, opts *subComma
...
-	cvd.ConnStatus, err = ConnectDevice(flags.CreateCVDOpts.Host, cvd.Name, &command{c, &flags.Verbose}, opts)
-	statePrinter.PrintDone(connectCVDStateMsg)
-	if err != nil {
-		err = fmt.Errorf("Failed to connect to device: %w", err)
+	for _, cvd := range cvds {
+		statePrinter.Print(fmt.Sprintf(connectCVDStateMsgFmt, cvd.Name))
+		cvd.ConnStatus, err = ConnectDevice(flags.CreateCVDOpts.Host, cvd.Name, &command{c, &flags.Verbose}, opts)
+		statePrinter.PrintDone(fmt.Sprintf(connectCVDStateMsgFmt, cvd.Name))
+		if err != nil {
+			err = fmt.Errorf("Failed to connect to device: %w", err)
```
Comment (jemoreira) [inline pkg/cli/cli.go:582]: This overwrites previous errors, multierror makes sense here.

File: pkg/cli/cli.go [hunk 3]
```diff
@@ -569,18 +569,22 @@ func runCreateCVDCommand(c *cobra.Command, flags *CreateCVDFlags, opts *subComma
...
+		}
+	}
+	for _, cvd := range cvds {
```
Comment (jemoreira) [inline pkg/cli/cli.go:585]: Any reason this is done in a separate loop?

Comment (ser-io) [reply pkg/cli/cli.go:586]: yes, form a UX perspective I'd like to see the order of events printed first and then the list of devices, e.g

```
Creating Host ... Done
Creating CVD ... Done
Connecting to cvd-1 ... Done
Connecting to cvd-2 ... Done

cvd-1 (cf-c8774d61-9197-4c1d-9f4f-3b4dd4bcd510)
  Status: Running
  ADB: connected
  Displays: [720 x 1280 ( 320 )]
  WebRTCStream: http://foo.com/client.html
  Logs: http://foo.com/logs/

cvd-2 (cf-c8774d61-9197-4c1d-9f4f-3b4dd4bcd510)
  Status: Running
  ADB: connected
  Displays: [720 x 1280 ( 320 )]
  WebRTCStream: http://foo.com/client.html
  Logs: http://foo.com/logs/
```

**Review by jemoreira — APPROVED** [review]

**Pull request merged**